packages/react-native/React/Base/RCTConstants.h

```diff
@@ -12,7 +12,7 @@ RCT_EXTERN NSString *const RCTPlatformName;
 RCT_EXTERN NSString *const RCTUserInterfaceStyleDidChangeNotification;
 RCT_EXTERN NSString *const RCTUserInterfaceStyleDidChangeNotificationTraitCollectionKey;
 
-RCT_EXTERN NSString *const RCTRootViewFrameDidChangeNotification;
+RCT_EXTERN NSString *const RCTWindowFrameDidChangeNotification;
```

Comment:
😞  This is breaking... 👮‍♂️

Reply:
Mentioned in a separate comment, but I think the original change only made it to 0.73 rc0. If it pick this to the 0.73 branch and then this changes only in the RC phase, would it still be breaking?